cmd/interactive.go

```diff
@@ -433,7 +443,7 @@ func generateInteractive(cmd *cobra.Command, opts runOptions) error {
 			sb.WriteString(line)
 		}
 
-		if sb.Len() > 0 && multiline == MultilineNone {
+		if sb.Len() > 0 && strings.TrimSpace(sb.String()) != "" && multiline == MultilineNone {
```

Comment:
This catches the case where a user has used `Ctrl-J` and then hits `Enter` but they still haven't actually typed in any text.